src/executorlib/task_scheduler/file/backend.py

```
@@ -45,15 +45,22 @@ def backend_write_file(file_name: str, output: Any, runtime: float) -> None:
+    try:
+        if "result" in output:
+            dump(
+                file_name=file_name_out + "_r.h5",
+                data_dict={"output": output["result"], "runtime": runtime},
+            )
+        else:
+            dump(
+                file_name=file_name_out + "_r.h5",
+                data_dict={"error": output["error"], "runtime": runtime},
+            )
+    except Exception as serialize_error:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)